volttron/platform/vip/agent/subsystems/ping.py

```diff
@@ -82,6 +82,8 @@ def ping(self, peer, *args):
         result = next(self._results)
         args = list(args)
         args.insert(0, b'ping')
+        if peer == b'pubsub':
+            peer = b''
```

Comment:
I think we can remove test_can_ping_pubsub as that test works on the assumption that pubsub is implemented as a service agent and this is no longer the case. In fact, any test that is working on that assumption should be removed. Then we can remove this check as it's really not needed.